libs/tup-components/src/tickets/TicketCreateModal.tsx

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
-import { useParams } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 import { Modal, ModalHeader, ModalBody } from 'reactstrap';
+import { TicketCreateForm } from './TicketCreateForm';
+import { useProfile } from '@tacc/tup-hooks';
+import './TicketCreateModal.scss';
```

Comment:
Requests:

1. Rename to `TicketCreateModal.global.css`. [Reference.](https://github.com/TACC/tup-ui/blob/task/tup-ui-styles/apps/tup-ui/src/styles/README.md)
2. Convert SCSS syntax—if any—to native CSS or  [PostCSS syntax](https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env#glossary).